its/src/test/java/com/sonar/it/shared/ScannerCliTest.java

```diff
@@ -30,19 +30,17 @@
 
 /**
  * Regression tests for scanning projects with the scanner-cli.
- * <p>
- * Note that this uses a different orchestrator instance than {@link com.sonar.it.csharp.Tests} or {@link com.sonar.it.vbnet.Tests}
  */
 @ExtendWith(Tests.class)
-public class ScannerCliTest {
+class ScannerCliTest {
```

Comment:
I removed the `public` access modifier because [S5786](https://sonarsource.github.io/rspec/#/rspec/S5786) was raising.
Same for the methods.